### DISCUSSION TITLE
Problem in site generating

### DISCUSSION THREAD

**Discussion by hatamiarash7** [upptime/upptime#451] [Q&A] [answered]
Hey there. There are some npm errors in the `Setup CI` and `Static Site CI` actions.

You can check [here](https://github.com/hatamiarash7/MyWebSite_Status/runs/3792347741?check_suite_focus=true#step:7:38) and [here](https://github.com/hatamiarash7/MyWebSite_Status/runs/3792347741?check_suite_focus=true#step:7:50)

I can't access my status page anymore. Can you help me?

**Comment by nijel** [comment]
Unfortunately, the error does not make the action fail - it still did commit and pushed the `gh-pages` branch and removed all the content.

The first error seems to be the package installation:

```
npm ERR! Could not install from "node_modules/semantic-release-gitmoji" as it does not contain a package.json file.
```

As a workaround, you can revert the latest deploy commit in the `gh-pages` branch. It will be alive until the `Static Site CI` is triggered again.

**Reply by hatamiarash7** [reply]
Unfortunately, this problem already existed. Before this, the website came up but did not show anything and an error of 500 appeared. It doesn't work at all now.

**Reply by nijel** [reply]
Ah, for me, it broke tonight and reverting the deploy commit brought it back (https://github.com/weblate-status-bot/status/commit/ac10d15afa051ef9f43204900ed662fac453eb40).

**Reply by AnandChowdhary** [reply]
We've opened an issue here: https://github.com/momocow/semantic-release-gitmoji/issues/32

**Comment by hatamiarash7** [comment, marked as answer]
This problem seems to be solved
